### PR TITLE
[Web Onboarding Project Task 2] Supabase Integrations

### PR DESCRIPTION
**Pull request by nehaahussain**
[//]: # "Feel free to customize this template to your project's vibes"

## What's new in this PR
Populated the post screen with data from the supabase table. Also includes the updated code from addressed comments in task 1.

## Screenshots
Screenshot 2024-10-19 at 7 57 43 PM

## Next steps
The output is the loading screen for less than a second then just a blank screen. I've tried debugging but can't seem to figure out the issue. Just wanted to create a PR of the progress so far.

Update: I fixed this issue by changing `<Img` to `<img`

CC: @ronniebeggs

### CODE REVIEW

File: supabase/client.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { createClient } from '@supabase/supabase-js';
+
+const supabase = createClient(
+  process.env.PUBLIC_SUPABASE_URL ?? '',
+  process.env.PUBLIC_SUPABASE_ANON_KEY ?? '',
+);
```
Comment (ronniebeggs) [inline supabase/client.ts:6]: Try adding the `NEXT_` prefix to each of your environment variables. I should've pointed this out earlier, but it seems that environment variables are only accessible to NextJs applications running in the browser if this prefix is includes ([docs here](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#bundling-environment-variables-for-the-browser)).

File: app/styles.module.css [hunk 2]
```diff
@@ -11,6 +11,106 @@
...
+
+.postInner {
+  display: flex;
+  align-items: center;
+  width: 100%;
+}
+
+.likesText {
+  margin-left: 8px;
+}
```
Comment (ronniebeggs) [inline app/styles.module.css:98]: This is one way to offset the like and username text. For flex boxes, we can alternatively apply a `column-gap` attribute, which I believe is a bit cleaner.

File: app/styles.module.css [hunk 3]
```diff
@@ -11,6 +11,106 @@
...
+
+.comment {
+  color: #262626;
+  font-family: Poppins;
+  font-size: 13px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 138.462%;
+  padding-bottom: 3%;
```
Comment (ronniebeggs) [inline app/styles.module.css:87]: Just out of curiosity, where did these values come from?

**Review by ronniebeggs — CHANGES_REQUESTED** [review]
Once you fix your environment variables, you should be able to see supabase data imported into your project. You're the right track! Good job!

File: app/styles.module.css (hunk 3 above)
Comment (nehaahussain) [reply app/styles.module.css:87]: `line-height: 138.462%;` is from the figma and `padding-bottom: 3%;` is from chatgpt.

**Pull request merged**